cli/connect/const.go

```diff
@@ -0,0 +1,89 @@
+package connect
+
+// setLanguagePrefix is used to set a language in the Tarantool console.
+const setLanguagePrefix = "\\set language"
+
+// setLanguage is a command to set a current language.
+const setLanguage = setLanguagePrefix
```

Comment:
The first is the command to send to `Tarantool` to change the language, the second is the command to change the language to `tt`. In fact it is the same string, but in theory it could be different strings.